Revert "fix(boto): specify the region while getting s3 connection"

bacongobbler opened the pull request:
This reverts commit 388c593dd182755fad2218ef2c1f24d39bc29948.

While it is the correct thing to do, it is a low severity issue and blocks users from upgrading to v2.9.0 due to #173. Reverting this commit returns the database to looking up and creating buckets in us-east-1. While it isn't optimal, it's not a security/billing issue.

Commit:
Revert "fix(boto): specify the region while getting s3 connection"

This reverts commit 388c593dd182755fad2218ef2c1f24d39bc29948.

kmala commented:
If this fix is reverted, then new installs doesn't work because we create the bucket in the us-east-1(default region) but the wal-e fails because we pass it a different/actual region passed by the user.

Commit:
Revert "fix(setup-envdir): convert AWS_REGION to WALE_S3_ENDPOINT"

This reverts commit 758632dae206685916fd2edeeb1113b9b455b86a.

bacongobbler commented:
Thanks, I just reverted that change as well.

Commit:
fix(setup-envdir): hardcode region to us-east-1

This forces wal-e to always use us-east-1 until we figure out a fix for
deis/postgres#173.

bacongobbler commented:
Also I've added a commit that hardcodes the region to us-east-1 as a stopgap until we fix this in the next release.

bacongobbler commented:
this won't work as it misses out the use case where users manually create the buckets in the correct region, so I'm closing this in favour of a better patch.
